src/JobDescription.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import Container from 'react-bootstrap/lib/Container';
+import withUserValidation from './withUserValidation';
+import { createBrowserHistory } from 'history';
+import jobUtils from './jobUtils';
+import './JobDescription.css';
+
+class JobDescription extends Component {
+
+  render() {
+    const history = createBrowserHistory();
+    const job = history.location.state.job;
```

Comment:
I think we should just fetch the job on mount. Otherwise we can't deep link to this page--this only works when navigating from job list.